axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveToolService.java

```diff
@@ -191,7 +191,7 @@ boolean isTemporarilyClosurePeriodManage(Period period, Journal journal, User us
 
   boolean checkMoveLinesCutOffDates(Move move);
 
-  boolean checkMoveOriginIsDuplicated(Move move) throws AxelorException;
+  List<Move> checkMoveOriginIsDuplicated(Move move) throws AxelorException;
```

Comment:
Maybe the method name should be changed as well, to something like `getMovesWithDuplicatedOrigin`